src/Providers/LaravelCaffeineService.php

```diff
@@ -30,7 +29,24 @@ public function provides() : array
         return ['genealabs-laravel-caffeine'];
     }
 
-    private function middlewareGroupExists(string $group) : bool
+    /**
+     * If the programmer defined something other than null in the configuration file we will use their config.
+     * Otherwise we will check if the web middleware exists, if not we will have no middleware.
+     *
+     * @return array
+     */
+    protected function getRouteGroupOptions() : array
+    {
+        $middleware_config = app(Cache::class)->get('genealabs-laravel-caffeine.middleware');
+
+        if ($middleware_config) {
+            return ['middleware' => $middleware_config];
+        }
+
+        return $this->middlewareGroupExists('web') ? ['middleware' => 'web'] : [];
+    }
+
+    protected function middlewareGroupExists(string $group) : bool
```

Comment:
I have a dislike for using `private` unless it is REALLY needed. In this case someone _may_ want to extend this class and would then be prevented from using the `middlewareGroupExists` method for no reason, as far as I can see.